students/9/main.js

```diff
@@ -51,8 +54,16 @@ const shortestString= (array) => {
    return retval[array.length- 1]
 };
 
-const main = async () => {
-  ex3();
+const countLetters = (array) => {
+    let letterCount = 0
+    for (let i = 0; i < 4; i++) {
```

Comment:
The number 4 in this case is an example of hard coding.  We cannot do this.  Try replacing this with `array.length`.